mercury_app/utils.py

```diff
@@ -192,3 +197,91 @@ def get_events_for_organizations(organizations, user):
         for e in event:
             e['org_name'] = organization['name']
     return event
+
+
+def create_webhook_from_view(user):
+    if not UserWebhook.objects.filter(user=user).exists():
+        token = get_auth_token(user)
+        webhook_id = create_webhook(token)
+        UserWebhook.objects.create(
+            user=user,
+            webhook_id=webhook_id,
+        )
+
+
+def create_webhook(token):
+    url = settings.URL_LOCAL
+    data = {
+        "endpoint_url": url + "/webhook-point/",
+        "actions": "order.placed",
+    }
+    response = Eventbrite(token).post('/webhooks/', data)
+    return (response[u'id'])
+
+
+def delete_webhook(token, webhook_id):
+    webhook = UserWebhook.objects.get(webhook_id=webhook_id)
+    webhook.delete()
+    Eventbrite(token).delete('/webhooks/' + webhook_id + "/")
```

Comment:
Probably you want to delete "your" local webhook after ensure it was deleted at EB, then I would call the EB api first.